test/test_api/.tmp_api/runhistory_B.json

```diff
@@ -0,0 +1,1815 @@
+{
```

Comment:
This file seems to be accidentally included, the .gitignore probably needs to be updated.

Reply:
Actually, we need this file for some tests. This ensures that we don't have to run `search` again and again to test various components. For example, the tests included [here](https://github.com/automl/Auto-PyTorch/blob/development/test/test_api/test_results_manager.py).